src/pages/projectSettings/tabs/PluginsTab/getFormSchema.ts

```diff
@@ -118,6 +118,8 @@ export const getFormSchema = (
       "ui:ObjectFieldTemplate": CardFieldTemplate,
       perfEnabled: {
         "ui:widget": widgets.RadioBoxWidget,
+        "ui:description":
+          "Enable the performance plugin (this requires the project to have matching ID and identifier).",
```

Comment:
I don't understand this description. Aren't the project Id and Identifier always different?

Reply:
No, you're able to create a project where you specify the ID and identifier match (because this is necessary for performance). This is indicated in the modal: 
![image](https://user-images.githubusercontent.com/26798134/169544632-24099322-8e39-49fd-803c-61c54606a2bb.png)

Reply:
interesting, I didn't realize that people make it match sometimes.